src/software/thunderscope/gl/layers/gl_world_layer.py

```diff
@@ -97,18 +98,25 @@ def __init__(
         self.ball_graphic = GLSphere(
             parentItem=self, radius=BALL_MAX_RADIUS_METERS, color=Colors.BALL_COLOR
         )
-        self.robot_status_graphic = GLCircle(
-            parentItem=self, color=Colors.BREAKBEAM_TRIPPED_COLOR
-        )
         self.ball_kick_velocity_graphic = GLLinePlotItem(
             parentItem=self, color=Colors.SPEED_VECTOR_COLOR
         )
 
+        # Set depth value of field line graphics to -1 so that they are
+        # rendered beneath robots and other graphics
+        self.friendly_defense_area_graphic.setDepthValue(-1)
+        self.enemy_defense_area_graphic.setDepthValue(-1)
+        self.field_lines_graphic.setDepthValue(-1)
+        self.field_outer_boundary_graphic.setDepthValue(-1)
+        self.halfway_line_graphic.setDepthValue(-1)
+        self.goal_to_goal_line_graphic.setDepthValue(-1)
+        self.field_center_circle_graphic.setDepthValue(-1)
```

Comment:
Can we add new constants for all depth values so we can easily configure them in case we need a new value higher than -1, but lower than X that's used for a different graphic?

Reply:
Done